web/src/utils/money.test.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {formatMoney} from 'utils/money';
+
+describe('formatMoney', () => {
+  test('it should format money accurately', () => {
+    const moneyInput = {
+      currency: 'USD',
+      dollars: 400,
+      cents: 39,
+    };
+    const expectedValue = '$400.39';
+
+    expect(formatMoney(moneyInput)).toBe(expectedValue);
+  });
+
+  test('it should be round money to nearest 2 decimal places', () => {
+    const moneyInputs = [
+      {
+        currency: 'USD',
+        dollars: 89,
+        cents: 2020,
```

Comment:
this is actually different from what I was thinking:
value = dollars + cents / 1000_000_000, cents is 0~999_999_999

alternatively you could also define cents is 0~99. then add comments and assertion when you construct the money object

Reply:
Will go with defining cents is 0~99! :)